assets/js/theme/global/modal.js

```diff
@@ -196,12 +210,65 @@ export class Modal {
         this.$content.html('');
     }
 
+    setupFocusableElements(modalType) {
+        this.$lastPageFocusedElement = $(document.activeElement);
+
+        const $collection = focusableElements[modalType]();
+        $collection.each((index, element) => $(element).removeClass(focusedClass)); 
+        const $elementToFocus = $($collection.get(0));
+        $elementToFocus.focus().addClass(focusedClass);
+
+        $('#modal').on('keydown', this.onTabbing.bind(null, modalType));
+    }
+
+    onTabbing(modalType, event) {
+        const isTAB = event.which === TAB;
+
+        if (!isTAB) return;
+
+        const $collection = focusableElements[modalType]();
+        const collectionLastIdx = $collection.length - 1
+
+        const $currentElement =  $collection.filter(`.${focusedClass}`);
```

Comment:
Better to use "each" to get the element and index during one iteration

Reply:
updated code. And now it is even not necessary performance will be little bit better